sdk/ai/azure-ai-projects/samples/evaluations/agentic_evaluators/sample_relevance.py

```diff
@@ -79,8 +79,8 @@ def main() -> None:
             print("Creating Eval Group")
```

Comment:
There are still several places that use the term "Eval Group". It would be good to update them all to "evaluation" for consistency.